apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.sln

```diff
@@ -20,7 +20,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Google.Cloud.Spanner.Data.T
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Google.Cloud.Spanner.Data.Snippets", "Google.Cloud.Spanner.Data.Snippets\Google.Cloud.Spanner.Data.Snippets.csproj", "{EBF9F4E0-C115-4DF0-BD06-2951A88A65AB}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Google.Cloud.ClientTesting", "..\..\tools\Google.Cloud.ClientTesting\Google.Cloud.ClientTesting.csproj", "{9FABF89B-6D3F-43BE-A9B5-C5D75ADBBB4D}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Google.Cloud.ClientTesting", "..\..\tools\Google.Cloud.ClientTesting\Google.Cloud.ClientTesting.csproj", "{9FABF89B-6D3F-43BE-A9B5-C5D75ADBBB4D}"
```

Comment:
Just in case you were wondering why this is happening, it's discussed in https://github.com/dotnet/project-system/issues/1821

Reply:
cool thanks